Demo/Demo/ViewModels/BaseViewModel.swift

```diff
@@ -191,16 +191,16 @@ class BaseViewModel: ObservableObject, PayPalWebCheckoutDelegate, CardDelegate {
     private func captureOrderOnMerchantServer(result: CardResult) {
         Task {
             let captureResult = try? await DemoMerchantAPI.sharedService.caputureOrder(orderID: result.orderID)
-            let status = captureResult?.status ?? result.status
-            updateTitle("Order ID:\(result.orderID) status: \(status)\n \(String(describing: result.paymentSource))")
+            let status = captureResult?.status ?? ""
+            updateTitle("Order ID:\(result.orderID) status: \(status)")
```

Comment:
Can we remove the status from the demo app for now?

Reply:
I'm not sure why we would take it out Jax. It's a value returned from merchant server on capture or authorize. I will go back and check Android PR to see if I missed anything.

Reply:
Is this a different status that this one that was removed with the changelog entry in the Android PR? `Remove status property from CardResult`?

Reply:
oh lol. I see what you mean. Just use what's returned from server.

Reply:
Yeah the web PPCP team helped us out a bit here. Since the merchant has to call capture / authorize to complete the transaction, that information will be available to them. We print it here mostly because the info is still available.

Reply:
This [PR is a work in progress](https://github.com/paypal/Android-SDK/pull/143) but we're going to do the same thing on Android in [CardFragment.kt](https://github.com/paypal/Android-SDK/blob/4b3b7b9eaab6ae864023a21bd83f1a1d7ab0ff72/Demo/src/main/java/com/paypal/android/ui/card/CardFragment.kt#L223).

Reply:
Gotcha, that's helpful. So _it is_ a different status than what was in the Android PR which was the confusion. It wasn't super clear that it was a different status since it's in iOS but still needs to be added on Android so seemed like it was the same one that we removed.